### DISCUSSION TITLE
Unklarheiten bei der Bestimmung der Dosis

### DISCUSSION THREAD

**Discussion by ChristianFischerJena** [Digitaler-Impfnachweis/certification-apis#227] [Q&A]
Hallo liebes Impfnachweis-Team,

wir haben große Probleme die richtige Dosis nach dem neuen Implementierungsleitfaden zu bestimmen. Zur Übersicht haben wir einmal alle relevanten Fälle aufgelistet und die unklaren Fälle fett markiert. (Die Tabelle listet zur Vollständigkeit auch die offlabel-use Fälle mit Janssen auf, die man vielleicht ignorieren kann.)

|1. Ereignis | 2. Ereignis | 3. Ereignis | Zertifikats-Dosis | Nächste Impfung|
-- | -- | -- | -- | --
  |   |   | - | fällig
Im |   |   | 1/2 | fällig nach 3W
Janssen |   |   | 1/1 | fällig nach 4W
Im | Im |   | 2/2 | fällig nach 3M
Im | Janssen |   | 2/2 | fällig nach 3M
Janssen | Im |   | 2/1 | fällig nach 3M
Janssen | Janssen |   | **2/1** | fällig nach 3M
genesen | Im |   | 1/1 | fällig nach 3M 
genesen | Janssen |   | 1/1 | fällig nach 4W
Im | Im | Im | 3/3 | fertig
Im | Im | Janssen |  ?  |  offlabel-use
Im | Janssen | Im | **3/3** | fertig, da gleichwertig zu Janssen/Im/Im
Im | Janssen | Janssen | ?   |  offlabel-use
Im | Genesen | Im | **2/1**  | fertig
Im | Genesen | Janssen | **2/1**  | fertig
Janssen | Im | Im | 3/1 | fertig
Janssen | Im | Janssen |  ?  |  offlabel-use
Janssen | Janssen | Im | 3/1 | fertig
Janssen | Janssen | Janssen |  ? |  offlabel-use
Janssen | Genesen | Im | **2/1** | fertig
Janssen | Genesen | Janssen | **2/1** | fertig
genesen | Im | Im | 2/1 | fertig
genesen | Im | Janssen | 2/1 | fertig
genesen | Janssen | Im | 2/1 | fertig
genesen | Janssen | Janssen | 2/1 | fertig
genesen | Genesen | Im | 1/1 | fällig nach 3M
genesen | Genesen | Janssen | 1/1 | fällig nach 3M

(Im = Impfung mit nicht Janssen Impfstoff)

Könnte bitte kurz jemand sagen, ob wir den Leitfaden richtig interpretiert haben oder ggf. die Tabelle korrigieren?

Herzliche Grüße

**Comment by birkfrau** [comment]
Hallo zusammen,
wir lösen folgende Fälle anders:
- Im	Janssen  **2/2**	fällig nach 3M	--> je nach Grundimmunisierung **2/1 oder 3/3**, da J&J-Zweitimpfung als Booster zählt
- Im	Janssen	Im  **3/3**	fertig, da gleichwertig zu Janssen/Im/Im --> je nach Grundimmunisierung **2/1 oder 3/3**, da J&J-Zweitimpfung als Booster zählt
- Im	Janssen	Janssen **3/3** offlabel-use --> je nach Grundimmunisierung **2/1 oder 3/3**, da J&J-Zweitimpfung als Booster zählt

Hier ist auf jeden Fall Klärung von offizieller Stelle notwendig.
Vielen Dank!
